release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,73 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as graceful upgrades, and improved worker labeling.
+
+### Important upgrade notes
+
+- Due to Azure limitations, upgraded clusters do not support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
+
+---
+
+### azure-operator v2.9.0
+
+- **TODO**
+
+### Kubernetes v1.16.3
+
+- No change from the previous version
+
+### Calico v3.10.1
+
+- No change from the previous version
+
+### cert-exporter [v1.2.1](https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#121-2019-12-24)
+
+- No change from the previous version
+
+### chart-operator [v0.11.4](https://github.com/giantswarm/chart-operator/releases/tag/v0.11.4)
+
+- Now handling timeouts by pulling chart tarballs.
+
+### cluster-operator [v0.22.1](https://github.com/giantswarm/cluster-operator/releases/tag/v0.22.1)
+
+- Supported enabling the proxy protocol with AWS ELB.
```

Comment:
@axbarsan sorry for confusion, these two changes are actually part of included nginx IC App (I've updated notes in cluster-operator 0.22.1 GitHub release); nginx changes are already mentioned down there; so maybe here just mention upgrade to nginx IC App 1.3.0, or generic "- No change from the previous version" since there was no real change in cluster-operator itself

Reply:
No problem, @sslavic . Thank you for clearing it up 👍